mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraSubscriptionModule.java

```diff
@@ -26,13 +26,15 @@
 import org.apache.james.backends.cassandra.components.CassandraModule;
 import org.apache.james.backends.cassandra.components.CassandraTable;
 import org.apache.james.backends.cassandra.components.CassandraType;
+import org.apache.james.backends.cassandra.utils.CassandraConstants;
 import org.apache.james.mailbox.cassandra.table.CassandraSubscriptionTable;
 
 import com.datastax.driver.core.schemabuilder.SchemaBuilder;
 import com.google.common.collect.ImmutableList;
 
 public class CassandraSubscriptionModule implements CassandraModule {
 
+    public static final int PER_USER_CACHED_SUBSCRIPTIONS = 100;
```

Comment:
Because I want to be able to cache up to one hundred subscription per user...

I tend to increase this value if there's reason for me to expect large partition.